src/dao.cpp

```diff
@@ -628,6 +662,26 @@ namespace hypha
       m_documentGraph.updateDocument(updater, oldHash, document.getContentGroups());
    }
 
+   void dao::createdao(const eosio::name &dao_name)
+   {
+      require_auth(get_self());
+
+      Document dao(get_self(), get_self(), getDAOContent(dao_name));
+
+      // Create the settings document as well and add an edge to it
+      ContentGroups settingCgs{
+          ContentGroup{
+              Content(CONTENT_GROUP_LABEL, SETTINGS)},
+            //   Content(ROOT_NODE, readableHash(dao.getHash()))},  
+          ContentGroup{
+              Content(CONTENT_GROUP_LABEL, SYSTEM),
+              Content(TYPE, common::SETTINGS_EDGE),
+              Content(NODE_LABEL, "Settings")}};
```

Comment:
This needs some extra information in order to make it unique at the beginning, otherwise if it's kept empty its hash could collide with new dao's settings docs. I would suggest adding the dao's name.

Reply:
Is there a way to make sure the DHO name is unique here? Maybe if you try to create a DHO with the same name, that would clash with an existing hash, this should fail?